modules/ROOT/pages/database-administration/index.adoc

```diff
@@ -1,114 +1,80 @@
-:description: Introduction to managing multiple active databases with Neo4j.
-[[manage-databases-introduction]]
-= Introduction
+[database-administration]]
+= Database administration
 
-[[manage-databases-concepts]]
-== Concepts
-
-With Neo4j {neo4j-version} you can create and use more than one active database at the same time.
-
-DBMS::
-Neo4j is a Database Management System, or _DBMS_, capable of managing multiple databases.
+Neo4j is a Database Management System, or DBMS, capable of managing multiple databases.
 The DBMS can manage a standalone server, or a group of servers in a cluster.
 
-Instance::
```

Comment:
Do we still have these kinds of definitions somewhere now or are they just removed?

Reply:
We will add in a future PR - will link once we have a working complete glossary (allowing for hovering etc).